src/client/Client.cc

```diff
@@ -11305,7 +11305,9 @@ int Client::write(int fd, const char *buf, loff_t size, loff_t offset)
 #endif
   /* We can't return bytes written larger than INT_MAX, clamp size to that */
   size = std::min(size, (loff_t)INT_MAX);
-  int r = _write(fh, offset, size, buf, NULL, false);
+  bufferlist bl;
+  bl.append(buf, size);
+  int r = _write(fh, offset, size, bl);
```

Comment:
```suggestion
  int r = _write(fh, offset, size, std::move(bl));
```

Reply:
Oh, thanks for the heads up. I'll add move semantics to the APIs that have been affected

Reply:
@batrick [added](https://github.com/ceph/ceph/compare/dd01c3fdd398adc96e235aa43b989fdee6497894..d2699c4bcad979699d1d1b91a7fb53b010ec8dfb), PTAL.